tests/test_helpers.py

```diff
@@ -0,0 +1,72 @@
+import pandas as pd
```

Comment:
Initially, I was a bit confused about the goal of this file. Can we add a one-line docstring such as "Unit-test for the `helper` module"